sql/portal-data-common.sql

```diff
@@ -447,6 +447,53 @@ insert into Region (regionId, countryId, regionCode, name, active_) values (8107
+insert into Region (regionId, countryId, regionCode, name, active_) values (9021, 9, 'JP-21', '岐阜県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9022, 9, 'JP-22', '静岡県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9023, 9, 'JP-23', '愛知県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9024, 9, 'JP-24', '三重県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9025, 9, 'JP-25', '滋賀県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9026, 9, 'JP-26', '京都府', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9027, 9, 'JP-27', '大阪府', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9028, 9, 'JP-28', '兵庫県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9029, 9, 'JP-29', '奈良県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9030, 9, 'JP-30', '和歌山県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9031, 9, 'JP-31', '鳥取県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9032, 9, 'JP-32', '島根県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9033, 9, 'JP-33', '岡山県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9034, 9, 'JP-34', '広島県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9035, 9, 'JP-35', '山口県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9036, 9, 'JP-36', '徳島県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9037, 9, 'JP-37', '香川県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9038, 9, 'JP-38', '愛媛県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9039, 9, 'JP-39', '高知県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9040, 9, 'JP-40', '福岡県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9041, 9, 'JP-41', '佐賀県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9042, 9, 'JP-42', '長崎県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9043, 9, 'JP-43', '熊本県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9044, 9, 'JP-44', '大分県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9045, 9, 'JP-45', '宮崎県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9046, 9, 'JP-46', '鹿児島県', TRUE);
+insert into Region (regionId, countryId, regionCode, name, active_) values (9047, 9, 'JP-47', '沖縄県', TRUE);
```

Comment:
I think the regionIds need to be in ascending order here.  There are a couple of jumps.